src/sagas/activity/sagas.js

```diff
@@ -1,36 +1,58 @@
-import { put, take, call } from 'redux-saga/effects';
+import { put, call } from 'redux-saga/effects';
+import type { Realm } from 'realm';
 
 import { messagesAdded, AddNewMessageAction } from '../../actions/activity';
 import defaultDB from '../../services/database';
+import type { MessageJobType as DBMessage } from '../../services/database/schemata';
 import {
   convertFromDatabase,
   convertToDatabase,
 } from '../../utils/mapping/activity';
 import type { ActivityLogMessage } from '../../types/ActivityLogMessage';
-import { createDatabaseUpdateChannel } from '../database';
+import { getCurrentAccountId, currentAccountBasedUpdate } from '../accounts/sagas';
 
+/**
+ * @desc Function that creates Realm results fetching settings for specific account.
+ * @param {Realm} db Realm instance.
+ * @param {string|null} accountId Id of account to fetch settings or null.
+ * @return {Realm.Results<AccountSettings>|null} Realm results fetching settings for specified account or null if not applicable.
+ */
+export function buildAccountSettingsResults(db: Realm, accountId: string | null) {
```

Comment:
As far as I see it should be named another way, because it's not about settings 🙂 Please update docs above as well.